src/app/pages/StoryPage/index.stories.jsx

```diff
@@ -1,16 +1,44 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import { withKnobs, select } from '@storybook/addon-knobs';
 import { BrowserRouter } from 'react-router-dom';
 import WithTimeMachine from '#testHelpers/withTimeMachine';
 
 import { StoryPage } from '..';
 import mundoPageData from './fixtureData/mundo';
 import persianPageData from './fixtureData/persian';
 
-const styStories = storiesOf('Pages|Story Page', module);
+const withSecondaryColumnsKnob = pageData => storyFn => {
```

Comment:
Great idea to add the stories, made it really easy to review this PR, running the stories from chromatic 👍